CHANGELOG.txt

```diff
@@ -113,6 +118,14 @@
             - In the class 'EntityCellCustomField' :
                 - The class attribute '_EXTRA_RENDERERS' has been reworked.
                 - The method '_get_renderer()' has been removed.
+        # In 'creme_core.forms':
+            - 'EnumerableChoiceSet' is now a base class that requires an Enumerator instead of a model Field and an EnumerableRegistry.
+            - A new 'FieldEnumerableChoiceSet' inherits from 'EnumerableChoiceSet' and have all its previous features.
+            - New 'EnumerableSelectMultiple' that supports multiple selection.
+        # In 'creme_core.forms.entity_filter':
+            - 'EnumerableSelect' are now used for EntityFilter regular field conditions.
```

Comment:
Not breaking change isn't it ?

Reply:
For the entity_filter part I wonder... replacing a DynamicSelect by EnumerableSelect is a breaking change or not ?

Reply:
Hum probably ; add "instead of DynamicSelect" in the changelog line and it's OK for me.